tests/core/job/test_job.py

```diff
@@ -147,6 +147,19 @@ def test_status_job(task):
     assert job.is_skipped()
 
 
+def test_stacktrace_job(task):
+    submission = _SubmissionManagerFactory._build_manager()._create(task.id, task._ID_PREFIX, task.config_id)
+    job = Job("job_id", task, submission.id, "SCENARIO_scenario_config")
+
+    fake_stacktrace = """Traceback (most recent call last):
+File "<stdin>", line 1, in <module>
+ZeroDivisionError: division by zero""".splitlines()
+
+    job.stacktrace = fake_stacktrace
+    assert len(job.stacktrace) == 3
+    assert all(st == fake_stacktrace[idx] for idx, st in enumerate(job.stacktrace))
```

Comment:
Looks like evidence. Why don't you run a function?

Reply:
I don't understand... What is an evidence?

Reply:
Ok, let's just simplify this code ;-)

Reply:
What I mean is that you define a variable and check whether the variable is correctly set. Why not create a job that raised this error and then check if the stacktrace is correctly set?

Reply:
That is the whole point of this PR: we need to set a stack trace that does not directly come from the execution of the job. In my case, the execution can/will be remote.

Reply:
oh my bad!